docs/docs-content/troubleshooting/generate-har-files.md

```diff
@@ -0,0 +1,155 @@
+---
+sidebar_label: "Generate a HAR File"
+title: "Generate HTTP Archive Files for Troubleshooting"
+description: "Learn how to generate and sanitize HAR files to speed up issue resolution."
+sidebar_position: 70
+tags: ["troubleshooting", "har", "support"]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCircleStop, faTrashCan } from '@fortawesome/free-regular-svg-icons';
+import { faBan, faPause, faPlay, faGear } from '@fortawesome/free-solid-svg-icons';
+import RecordNetworkLog from '/src/components/CustomIcons/RecordNetworkLog';
+
+When you encounter page performance or rendering issues in Palette, we may ask you to generate an HTTP Archive (HAR) file to help us identify and resolve those issues.
+
+:::caution
+HAR files contain all network activity for a specific page, including sensitive and confidential information, such as API keys, secrets, cookies, passwords, and more. Before sending your HAR files to us, sanitize the sensitive data. We do not recommend using any third-party tools for HAR sanitization, as they can overlook some details.
+:::
+
+This guide explains how you can generate HAR files in Chrome, Safari, and Firefox and sanitize the sensitive data they contain.
+
+## Generate HAR Files
+
+<Tabs>
+  <TabItem value="chrome" label="Chrome" default>
+    1. Open the page with issues in Chrome.
+    2. On your keyboard, open Developer Tools:
+       - For Windows and Linux, press **F12** or **Ctrl + Shift + I**.
+       - For Mac, press **Fn + F12** or **Cmd + Option + I**.
+
+       For alternative methods to open Developer Tools, refer to [Open Chrome DevTools](https://developer.chrome.com/docs/devtools/open).
+    3. In Developer Tools, select the **Network** tab. Then, on the **Network** pane:
+       - Toggle <RecordNetworkLog /> to <FontAwesomeIcon icon={faCircleStop} color={"#E46962"} /> to start recording network activity.
+       - Click <FontAwesomeIcon icon={faBan} rotation={270} /> to clear the current network log.
+       - Select the **Preserve log** and **Disable cache** checkboxes.
+
+       ![](/assets/docs/images/troubleshooting/generate-har-files/chrome-dev-tools-settings.png)
+    4. On the page, reproduce the issues you've encountered.
+    5. In the network log, right-click any item and select **Save all as HAR with content**.
+  </TabItem>
+  <TabItem value="safari" label="Safari">
+    1. If you haven't yet, [enable the web development features in Safari](https://developer.apple.com/documentation/safari-developer-tools/enabling-developer-features).
+    2. Open the page with issues.
+    3. On your keyboard, press **⌥ + ⌘ + I** to open Web Inspector.
```

Comment:
What about Windows?

Reply:
I didn't include Windows here because it feels like Windows users wouldn't use Safari.